wicket-core/pom.xml

```diff
@@ -207,6 +205,22 @@ org.apache.wicket.validation.validator;-noimport:=true
 				<groupId>org.primefaces.extensions</groupId>
 				<artifactId>resources-optimizer-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-jar-plugin</artifactId>
+				<executions>
+					<execution>
+						<goals>
+							<goal>test-jar</goal>
```

Comment:
As I said above I think this should not be needed.

Reply:
In this version it is required, in the alternative one not, see https://github.com/apache/wicket/pull/660.